easybuild/easyconfigs/g/Ghostscript/Ghostscript-9.16-goolf-1.7.20.eb

```diff
@@ -0,0 +1,30 @@
+easyblock = 'ConfigureMake'
+
+name = 'Ghostscript'
+version = '9.16'
+
+homepage = 'http://ghostscript.com'
+description = """Ghostscript is a versatile processor for PostScript data with the ability to render PostScript to
+ different targets. It used to be part of the cups printing stack, but is no longer used for that."""
+
+toolchain = {'name': 'goolf', 'version': '1.7.20'}
+toolchainopts = {'pic': True}
+
+source_urls = ["http://downloads.ghostscript.com/public/"]
+sources = [SOURCELOWER_TAR_GZ]
+
+dependencies = [
+    ('freetype', '2.5.3'),
+    ('fontconfig', '2.11.1'),
+    ('GLib', '2.40.0'),
+    ('libjpeg-turbo', '1.3.1'),
+    ('libpng', '1.6.12'),
```

Comment:
The above would be nice to become aligned with common_deps #1689.
ie. bump forward libpng; or, does that bring up dependency hell with other deps? if so, it's a pass from me...